external/kafka-0-10-sql/src/main/scala/org/apache/spark/sql/kafka010/KafkaOffsetReader.scala

```diff
@@ -115,7 +116,7 @@ private[kafka010] class KafkaOffsetReader(
   def fetchTopicPartitions(): Set[TopicPartition] = runUninterruptibly {
     assert(Thread.currentThread().isInstanceOf[UninterruptibleThread])
     // Poll to get the latest assigned partitions
-    consumer.poll(0)
+    consumer.poll(JDuration.ofMillis(0))
```

Comment:
Could you revert these changes? We don't use java.time.Duration in Spark.

Reply:
Depending on the Kafka release we agree upon, I can revert.
Duration is recommended API for 2.0.0 release

Reply:
@zsxwing Why do you want to support Kafka clients jars from 0.10 to 2.0? Since newer clients jars support older brokers, we recommend people use the latest Kafka clients jar whenever possible.